SAMtags.tex

```diff
@@ -234,13 +239,36 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
+Barcode sequence (Identifying the sample/library), with any quality scores stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+What one does with multiple sample barcodes (e.g. one on each end of the template) is left to the implementation. 
+One current implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
```

Comment:
I would change this from "one current implementation" to "the recommended implementation is to" since that will promote consistency, and that's already how multiple toolkits are doing it.